### PR TITLE
UNR-5126 fix WellKnownEntity server crash

### PR DESCRIPTION
**Pull request by simonsarginson**
Fix for NFR crash

### CODE REVIEW

**Review by jacquese — APPROVED** [review]

**Comment by m-samiec** [comment]
https://improbableio.atlassian.net/browse/UNR-5126

**Review by m-samiec — APPROVED** [review]

File: SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialNetDriver.cpp [hunk 1]
```diff
@@ -865,7 +865,7 @@ void USpatialNetDriver::OnMapLoaded(UWorld* LoadedWorld)
 		return;
 	}
 
-	if (IsServer())
+	if (IsServer() && WellKnownEntitySystem != null)
```
Comment (simonsarginson) [inline SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialNetDriver.cpp:868]:
```suggestion
	if (IsServer() && WellKnownEntitySystem != nullptr)
```

File: SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialNetDriver.cpp [hunk 2]
```diff
@@ -865,7 +865,7 @@ void USpatialNetDriver::OnMapLoaded(UWorld* LoadedWorld)
 		return;
 	}
 
-	if (IsServer())
+	if (IsServer() && WellKnownEntitySystem != nullptr)
```
Comment (martin-improbable) [inline SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialNetDriver.cpp:868]: IsValid() would have been more consistent with similar usage elsewhere in this file?

Comment (simonsarginson) [reply SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialNetDriver.cpp:868]: I see both being used interchangeably maybe something for a styleguide discussion

Comment (simonsarginson) [inline SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialNetDriver.cpp:868]:
```suggestion
	if (IsServer() && WellKnownEntitySystem.isValid())
```

File: SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialNetDriver.cpp [hunk 3]
```diff
@@ -865,7 +865,7 @@ void USpatialNetDriver::OnMapLoaded(UWorld* LoadedWorld)
 		return;
 	}
 
-	if (IsServer())
+	if (IsServer() && WellKnownEntitySystem.isValid())
```
Comment (simonsarginson) [inline SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialNetDriver.cpp:868]:
```suggestion
	if (IsServer() && WellKnownEntitySystem.IsValid())
```

**Pull request merged**